Sources/FrenchRepublicanCalendarCore/DecimalTime.swift

```diff
@@ -69,7 +69,15 @@ public struct DecimalTime {
 public extension DecimalTime {
     /// Initializes a new DecimalTime with the current time
     init(base: Date = Date()) {
-        let midnight = Calendar.gregorian.startOfDay(for: base)
+        self.init(base: base, timeZone: nil)
+    }
+
+    init(base: Date = Date(), timeZone: TimeZone?) {
+        var GregorianCalendar: Calendar = Calendar.gregorian
```

Comment:
Use the [Swift naming conventions](https://www.swift.org/documentation/api-design-guidelines/#general-conventions): lowerCamelCase for variables